docs/network/evm-nodes.md

```diff
@@ -140,6 +188,31 @@ The following examples use `<SR_OBSERVER_DATA_DIR>` as the location of this temp
 
    The `--initial-kernel` argument is needed only the first time that you start the node.
 
+## Switching history modes
+
+After the node is running in one mode you can switch to another history mode:
+
+1. Stop the node.
+1. Switch to the new mode with the command `octez-evm-node switch history to <MODE>`, where `<MODE>` is the new mode.
+1. Start the node as usual with the command `octez-evm-node run`.
+
+For example, you can switch to `rolling` mode with the command `octez-evm-node switch history to rolling:<DAYS>`, where `<DAYS>` is the number of days of compete data to keep.
+Then when you restart the node, run `octez-evm-node run observer --network testnet --history rolling:<DAYS> --data-dir <EVM_DATA_DIR>`.
+
+Because the node cannot reconstruct data that it does not have, you can switch from only some modes to some other modes:
+
+- The node can go from `archive` mode to `rolling` or `full` mode.
+- The node can go from `full` mode to `rolling` mode or `full` mode with a different retention period.
+- The node can go from `rolling` mode to `rolling` mode with a different retention period.
+
+When you switch from `full` or `rolling` mode and change the retention period, the node can increase the retention period, but it does not immediately retrieve the new data.
+For example, you can switch from `rolling:1` mode to `rolling:5` mode, but the node does not immediately download the previous 5 days of data.
+Instead, it begins storing more data as time passes until it has enough for 5 days of history.
+You can also start a node in `rolling:5` mode by downloading a rolling snapshot that is 5 days old or by switching from `archive` mode or `full` mode.
```

Comment:
A `full` node won’t give you more states than a `rolling` node.

Reply:
Changed to "or by switching from `archive` mode."